pyomo/contrib/mindtpy/tests/MINLP2_simple.py

```diff
@@ -70,7 +70,7 @@ def __init__(self, *args, **kwargs):
         m.const7 = Constraint(expr=Y[1] + Y[2] <= 1)
 
         """Cost (objective) function definition"""
-        m.cost = Objective(expr=-5*Y[1] - 6*Y[2] - 8*Y[3] - X[4], sense=maximize)
+        m.cost = Objective(expr=+5*Y[1] + 6*Y[2] + 8*Y[3] + X[4], sense=minimize)
```

Comment:
Come to think of it, I can change it back so that we check for maximization as well.

Reply:
Fixed in  5a9a61c446342e6649022595553a54a2b1e68d3e